doc/user_manual/model.tex

```diff
@@ -1432,3 +1432,271 @@ \subsection{EnsembleModel}
+  this specific entity from other input blocks in the XML.
+  \item \xmlAttr{subType}, \xmlDesc{required string attribute}, must be kept
+  empty.
+  %
+\end{itemize}
+\vspace{-5mm}
+
+Within the \xmlNode{HybridModel} XML node, the multiple entities that constitute
+this HybridModel needs to be inputted. 
+
+\begin{itemize}
+  \item \xmlNode{Model}, \xmlDesc{XML node, required parameter}.
+  %
+  The text portion of this node needs to contain the name of the Model
+  %
+  \\This XML node needs to contain the attributes:
+
+\vspace{-5mm}
+\begin{itemize}
+  \itemsep0em
+  \item \xmlAttr{class}, \xmlDesc{required string attribute}, the class of this sub-model (e.g. Models)
+  %
+  \item \xmlAttr{type}, \xmlDesc{required string attribute}, the sub-type of this Model (e.g. ExternalModel, ROM, Code)
+  %
+\end{itemize}
+\vspace{-5mm}
+
+  \item \xmlNode{ROM}, \xmlDesc{XML node, required parameter}.
+  %
+  The text portion of this node needs to contain the name of the ROM
+  The user can specify as many \xmlNode{ROM} as required by the \xmlNode{Model}.
+  \nb The outputs of each ROM should be different, and the total set of ROMs' outputs
+  should be the same as the set of \textit{Model's} outputs.
+  %
+  \\This XML node needs to contain the attributes:
+
+\vspace{-5mm}
+\begin{itemize}
+  \itemsep0em
+  \item \xmlAttr{class}, \xmlDesc{required string attribute}, the class of this sub-model (e.g. Models)
+  %
+  \item \xmlAttr{type}, \xmlDesc{required string attribute}, the sub-type of this Model (e.g. ROM)
+  %
+\end{itemize}
+\vspace{-5mm}
+
+  \item \xmlNode{CV}, \xmlDesc{XML node, required parameter}.
+  %
+    The text portion of this node needs to contain the name of the \xmlNode{PostProcessor} with \xmlAttr{subType}
+    ``CrossValidation``.
+  %
+  \\This XML node needs to contain the attributes:
+
+\vspace{-5mm}
+\begin{itemize}
+  \itemsep0em
+  \item \xmlAttr{class}, \xmlDesc{required string attribute}, the class of this sub-model (e.g. Models)
+  %
+  \item \xmlAttr{type}, \xmlDesc{required string attribute}, the sub-type of this Model (e.g. PostProcessor)
+  %
+\end{itemize}
+\vspace{-5mm}
+  
+  \item \xmlNode{TargetEvaluation}, \xmlDesc{XML node, required parameter}.
+  %
+    The text portion of this node needs to contain the name of a data object defined in the \xmlNode{DataObjects} block.
+    \nb currently only accept data object wih type ``PointSet``. The \xmlNode{TargetEvaluation} is primary used for
+    training ROMs. \nb The linked DataObject should be placed as additional output of the Step where the
+    \textbf{HybridModel} is used.
+  %
+  \\This XML node needs to contain the attributes:
+
+\vspace{-5mm}
+\begin{itemize}
+  \itemsep0em
+  \item \xmlAttr{class}, \xmlDesc{required string attribute}, the class of the DataObjects.
+  %
+  \item \xmlAttr{type}, \xmlDesc{required string attribute}, the sub-type of the DataObjects (e.g. ``PointSet``)
+  %
+\end{itemize}
+\vspace{-5mm}
+  %
+\end{itemize}
```

Comment:
Probably since the copy paste of the explanation here, you can just create a custom command whit a single attribute. See samplers.tex or steps.text

Reply:
fixed